### ISSUE TITLE
shasum of bottle mismatch

### ISSUE THREAD

**Issue by soonhokong** [leanprover/homebrew-lean#3]
Sometimes `update.sh` script puts a wrong shasum for the built bottle, which results in building from the source instead of pouring the bottle.

```
$ brew install lean
==> Installing lean from leanprover/homebrew-lean
==> Downloading https://leanprover.github.io/homebrew-lean/lean-0.2.0-gitb66b3a6c58ab8386c17a002ddaef32560383fcf5.yosemite.bottle.tar.gz
######################################################################## 100.0%
Error: SHA1 mismatch
Expected: 9352d891fb1512e1ba76036f428289a3e3c9c2c7
Actual: 5888bf73753d5e2ccf73d668ed8e581d4e0bad33
Archive: /Library/Caches/Homebrew/lean-0.2.0-gitb66b3a6c58ab8386c17a002ddaef32560383fcf5.yosemite.bottle.tar.gz
To retry an incomplete download, remove the file above.
Warning: Bottle installation failed: building from source.
==> Cloning https://github.com/leanprover/lean.git
Updating /Library/Caches/Homebrew/lean--git
==> Checking out branch master
==> cmake ../src -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/lean/0.2.0-gitb66b3a6c58ab8386c17a002ddaef32560383fcf5 -DCMAKE_BUILD_TYPE=Release -DEMACS_LISP_DIR=/usr/local/share/emacs/site-lisp/lean -DTCMALLOC=OFF -DLIBRARY_DIR=./
==> make -j2
```

**Comment by soonhokong** [comment]
I forked this repo and created https://github.com/soonhokong/homebrew-lean to fix and test the problem. So far it works fine. I've added a travis-ci support as well to test the lean formula everytime we update it. It works well as we can see in https://travis-ci.org/soonhokong/homebrew-lean/builds. It picks up the binary distribution and takes less than a minute to install lean and its dependent packages (mpfr, gmp, lua, ninja) on a OSX VM.

I have two OSX machines (one with 10.9 and another with 10.10) running cron jobs to update the formula periodically. Once I confirm that it is stable and `brew upgrade` picks up the new version, I'll apply the changes to `leanprover/homebrew-lean` repo, monitor things, and close this issue.
